docs/xcm/integration/zombienet-testing.md

```diff
@@ -0,0 +1,107 @@
+
+At the moment, only production networks, like `Astar` or `Shiden`, have access to a great number of HRMP channels with various parachains. Testing in production isn't a good idea though and should be avoided. A possible alternative is to use `Rococo` as relay chain and the connected parachains, if they are available. Not all projects deploy a testnet on Rococo.
+
+## Zombienet
+
+The approach that gives users the most control is to setup a local test network, with both the relay chain and parachain(s). This is very simple & easy to do using the [zombienet](https://github.com/paritytech/zombienet) tool provided by Parity. Simply put, users can download arbitrary relay-chain and parachain binaries (or use images) to setup a configurable local test network. Users will have access to all privileged actions on the relay-chain and on the parachains which makes testing much easier.
+
+For example, user can download `polkadot` binary together with `astar-collator` and `cumulus-parachain` to spin up a testnetwork with `polkadot` as relay chain, `astar` as one parachain and `statemint` as the second parachain.
+
+`Zombienet` documentation can be found [here](https://paritytech.github.io/zombienet/), including the installation instructions, CLI usage, guide with examples and many more. Users are advised to consult this document to get a better understanding of the tool.
+
+## Shibuya - Shiden Test Network
+
+### Overview
+
+A quick reminder - `Shibuya` is a test network with no market value used by `Astar & Shiden` team to test features before deploying into production. It uses a custom **Rococo** based relay chain. `Shiden` is a production canary-type network connected to `Kusama`. These two parachains aren't aware of one another and do not communicate in live networks.
+
+However, using `zombienet`, users can setup a local test network where one parachain would be `Shibuya` and the other would be `Shiden` with **HRMP** channels opened between them. This is incredibly useful for testing & integration because it gives users the option for cross-chain communication between two smart-contract oriented parachains which support WASM smart contracts, amongst many other features.
+
+The following instructions will explain how to setup & configure local _Shibuya - Shiden test network_.
+
+### Basic Setup Instructions
+
+For users who already know what they are doing, please check [this](https://github.com/AstarNetwork/Astar/tree/master/third-party/zombienet) folder in `Astar` repository for _ready-to-use_ `zombienet` configurations.
+
+1. For the sake of simplicity, prepare a folder called `zombienet` into which **ALL** binaries and config files will be placed.
+
+2. Download `zombienet` binary appropriate for your operating system (or install it in any way you prefer).
```

Comment:
```suggestion
2. [Download `zombienet` binary](https://github.com/paritytech/zombienet/releases) appropriate for your operating system (or install it in any way you prefer).
```

Reply:
I intentionally skipped this since I already linked zombienet repo above.